### ISSUE TITLE
Colour Contrast in Dashboard Issue

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#759]
**Describe the bug**
using wave extension, we are getting 9 contrast issues. When we are inspecting them, and fixing them with #666666 colour, then also they are not getting fixed

**To Reproduce**
1. Go to Dashboard (http://localhost:9992/cm/app/?view=dash&debug=true)
2. Run wave firefox extension.
3. Go to Details. And see contrast Errors.

**Expected behavior**
There should be no contrast errors.

**Additional Context**
The issue seems to be with styles.css (https://github.com/percussion/percussioncms/blob/b6cdc8697d3047c2d8c1ad0b5612319aeef8f75b/WebUI/war/css/styles.css)

**Comment by ramdev1980** [comment]
Unable to reproduce in chrome. It's a firefox issue.

**Comment by ramdev1980** [comment]
It seems that it got fixed while fixing other accessibility issues. So closing it as it is not reproducible now.
